### PR TITLE
Add luarocks and luabinaries

### LINKED ISSUE #1708: [Request] Add luarocks
A package manager for Lua modules.

[Official site](https://luarocks.org/)
[Project](https://github.com/luarocks/luarocks)
[Wiki(Download)](https://github.com/luarocks/luarocks/wiki/Download)

### PR DESCRIPTION
**Pull request by wsw0108**
- luarocks from official release
- lua5.3 from https://sourceforge.net/projects/luabinaries

try close #1708

### CODE REVIEW

File: bucket/lua-windows.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (rasa) [inline bucket/lua-windows.json:1]: There already is a [lua-for-windows](https://github.com/ScoopInstaller/Main/blob/master/bucket/lua-for-windows.json). Is it correct to have both? Is lua-windows the correct name, given the name of this other package?

Comment (wsw0108) [reply bucket/lua-windows.json]: `lua-for-windows` contains both lua5.1 and luarocks(may be 2.x), but the sources are not activate maintained.

**Comment by wsw0108** [comment]
Please use [https://github.com/wsw0108/scoop-bucket](https://github.com/wsw0108/scoop-bucket).

**Comment by rashil2000** [comment]
Can you make a PR for the lua-windows.json and luarocks.json from your bucket to this Main bucket?

**Pull request closed without merging**